geomstats/geometry/discrete_curves.py

```diff
@@ -1842,75 +1842,21 @@ def _init_space_with_reparametrizations(self, total_space):
         space.fiber_bundle = SRVReparametrizationBundle(space)
         return space
 
-    def horizontal_projection(self, tangent_vec, base_point):
-        """Project to horizontal subspace."""
-        raise NotImplementedError("Horizontal projection is not implemented.")
-
-    def align_rotation(self, point, base_point, return_rotation=False):
-        """Find optimal rotation of curve with respect to base curve.
-
-        Parameters
-        ----------
-        point : array-like, shape=[..., k_sampling_points - 1, ambient_dim]
-            Discrete curve to align.
-        base_point : array-like, shape=[..., k_sampling_points - 1, ambient_dim]
-            Reference discrete curve.
-        return_rotation : boolean
-            If true, returns the optimal rotation used for the alignment.
-            Optional, default : False.
-
-        Returns
-        -------
-        aligned : array-like, shape=[..., k_sampling_points - 1, ambient_dim
-            Curve optimally rotated with respect to reference curve.
-        """
-        return self._total_space_with_rotations.fiber_bundle.align(
-            point, base_point, return_rotation
-        )
-
-    def align_reparametrization(self, point, base_point, spline):
-        """Find optimal parametrization of a curve with respect to a base curve.
-
-        Parameters
-        ----------
-        point : array-like, shape=[..., k_sampling_points - 1, ambient_dim]
-            Discrete curve to align.
-        base_point : array-like, shape=[..., k_sampling_points - 1, ambient_dim]
-            Reference discrete curve.
-        spline : function
-            Spline function that interpolates between the points of the curve to
-            align.
-
-        Returns
-        -------
-        aligned : array-like, shape=[..., k_sampling_points - 1, ambient_dim
-            Curve optimally reparametrized with respect to reference curve.
-        """
-        bundle = self._total_space_with_reparametrizations.fiber_bundle
-        return bundle.aligner.discrete_horizontal_geodesic(base_point, point, spline)[
-            ..., -1, :, :
-        ]
-
     def _align_single(self, point, base_point):
-        """Align point to base point, non vectorized."""
-        aligned_point = gs.copy(point)
-        rotation = gs.eye(self._total_space.ambient_manifold.dim)
+        """Align point to base point."""
```

Comment:
Add docstring and parameter check.

Or better: add automatic `numpydoc` validation checks in another PR?

See how we did it in TopoModelX?
https://github.com/search?q=repo%3Apyt-team%2FTopoModelX%20numpydoc&type=code

Reply:
Added `numpydoc` to the todo list and updated docstring.